lib/kitchen/directions/bake_chapter_answer_key/strategies/calculus.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeChapterAnswerKey
```

Comment:
Since the end of chapter doesn't contain an answer key, could this module be renamed to something along the lines of `BakeChapterExercises`?